CommandConsole.cpp

```diff
@@ -420,3 +441,74 @@ HOOK_METHOD(MouseControl, OnRender, () -> void)
     PrintHelper::GetInstance()->Render();
     super();
 }
+
+HOOK_METHOD(InputBox, StartInput, () -> void)
+{
+    LOG_HOOK("HOOK_METHOD -> InputBox::StartInput -> Begin (CommandConsole.cpp)\n")
+    CommandConsole::GetInstance()->cursorPosition = 0;
+    super();
+}
+
+HOOK_METHOD(InputBox, TextEvent, (CEvent::TextEvent event) -> void)
+{
+    LOG_HOOK("HOOK_METHOD -> InputBox::TextEvent -> Begin (CommandConsole.cpp)\n")
+
+    hs_log_file("TextEvent Cursor position before: %d\n", CommandConsole::GetInstance()->cursorPosition);
+    hs_log_file("TextEvent Event: %d\n", event);
+    cursorTickCount = 0;
+    if (event == 3 && CommandConsole::GetInstance()->cursorPosition > 0)
```

Comment:
Could we add some comments describing the different event numbers, or use an enum so it's somewhat self-describing?

Reply:
Added the enum.